Added feedback after partner event (#558)
AdobeDocs/commerce-extensibility: src/pages/app-management/installation/webhooks.md

# Webhooks

> The `webhooks` field in your `app.commerce.config` file declares [Adobe Commerce webhook](../../webhooks/index.md) subscriptions for your application and configures the subscriptions on the merchant's Admin. Webhooks allow your app to make calls to third-party systems synchronously.

Reviewer: "...In the merchant's Admin."?

Author: ```suggestion
The `webhooks` field in your `app.commerce.config` file declares [Adobe Commerce webhook](../../webhooks/index.md) subscriptions for your application and configures the subscriptions in the merchant's Admin. Webhooks allow your app to make calls to third-party systems synchronously.
```

Use [Events](./events.md) and webhooks when you need both asynchronous event delivery and synchronous hooks inside Commerce processes.

## Responsibilities by role

Webhook configuration spans both the developer who ships the app and the merchant who associates it with Commerce:

* **App developers** declare webhooks in the `webhooks` field of `app.commerce.config`. That manifest is what App Management uses to know which webhook subscriptions belong to your app.

> * **Merchants** usually get webhooks from that manifest without extra setup. If **Adobe Commerce** or your app still needs more steps, see [Install and access App Management](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#access-app-management) and [Commerce webhooks and apps](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#commerce-webhooks-and-apps) (Experience League) — user guides in the Adobe Commerce Help site.

Reviewer: what does "need more steps" mean? Also, I don't think you need "(Experience League) — user guides in the Adobe Commerce Help site."

> * **Merchants** usually get webhooks from that manifest without extra setup. If **Adobe Commerce** or your app still needs more steps, see [Install and access App Management](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#access-app-management) and [Commerce webhooks and apps](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#commerce-webhooks-and-apps) (Experience League) — user guides in the Adobe Commerce Help site.

Author: ```suggestion
* **Merchants** usually get webhooks from that manifest without extra setup. See [Install and access App Management](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#access-app-management) and [Commerce webhooks and apps](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#commerce-webhooks-and-apps)  for more information on how to configure Adobe Commerce.
```

# Webhooks

> The `webhooks` field in your `app.commerce.config` file declares [Adobe Commerce Webhook](https://developer.adobe.com/commerce/extensibility/webhooks/) subscriptions for your application. App Management uses that definition during installation to provision the subscriptions your app needs, similar to how [Events](./events.md) subscriptions are driven from your configuration so merchants get an out-of-the-box experience instead of assembling webhook setup themselves. Webhooks allow your app to respond to live Commerce processes, such as checkout, or cart validation.

Reviewer 2: ```suggestion
The `webhooks` field in your `app.commerce.config` file declares [Adobe Commerce webhook](https://developer.adobe.com/commerce/extensibility/webhooks/) subscriptions for your application and configures the subscriptions on the merchant's Admin. Webhooks allow your app to make calls to third-party systems synchronously.
```

Also, the link needs to be a relative path, not hardcoded.

## Responsibilities by role

Webhook configuration spans both the developer who ships the app and the merchant who associates it with Commerce:

* **App developers** declare webhooks in the `webhooks` field of `app.commerce.config`. That manifest is what App Management uses to know which webhook subscriptions belong to your app.
* **Merchants** usually get webhooks from that manifest without extra setup. If **Adobe Commerce** or your app still needs more steps, see [Install and access App Management](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#access-app-management) and [Commerce webhooks and apps](https://experienceleague.adobe.com/en/docs/commerce/app-management/install#commerce-webhooks-and-apps) (Experience League) — user guides in the Adobe Commerce Help site.

### Use Events and webhooks together

> Use [Events](./events.md) and webhooks when you need both asynchronous event delivery and synchronous hooks inside Commerce processes.

Reviewer 2: Make this the last sentence of the intro section of this topic.
